helm/charts/keto/templates/job-migration.yaml

```diff
@@ -20,6 +20,9 @@ metadata:
     {{- with .Values.job.annotations }}
       {{- toYaml . | nindent 4 }}
     {{- end }}
+    helm.sh/hook-weight: "1"
```

Comment:
maybe instead of hardcoding them here, move them to the values.yaml like we have in [keto](https://github.com/ory/k8s/blob/master/helm/charts/keto/values.yaml#L41-L49=)